app/assets/stylesheets/layout.scss

```diff
@@ -0,0 +1,63 @@
+#outer-container, #topnav-container, #search-navbar-container, #sul-footer-container {
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  border: none;
+}
+
+.header-logo > a, h3.panel-title a {
+  border-bottom: none;
+}
+
+#topnav, #search-navbar, #main-container, #sul-footer, #masthead-container {
```

Comment:
Avoid using id selectors<br>Each selector in a comma sequence should be on its own line